### DISCUSSION TITLE
Custom Avatars not showing up? Broken??

### DISCUSSION THREAD

**Discussion by BonzoMartini** [mozilla/hubs#3412] [Hubs Cloud Questions]
**Description**
Adding Avatars broke existing avatars and doesn't work

**steps to repeat**
in Hubs Admin go to import content
Select the Hubs avatars pack file
Select preview import
Choose 1 avatar
Select import
Look in Featured Avatars - the avatar is there. (in this case the "Swirly Glow" avatar)
<img width="1388" alt="featured avatars = many" src="https://user-images.githubusercontent.com/66266175/99907314-0eec7b00-2cd4-11eb-95cc-9d3b683f971f.png">

Open a Hubs scene go to add an avatar
All is blank

<img width="680" alt="featured=blank" src="https://user-images.githubusercontent.com/66266175/99907309-0b58f400-2cd4-11eb-93e1-e55e0e3f289d.png">

**Expected behaviour**
These steps did work previously and I had added avatars
I have tried uploading glb files and importing packs 
Please explain like I'm 5.  A simple guide would help

**Comment by BonzoMartini** [comment]
I looked at this [video  ](https://www.youtube.com/watch?v=nQ85L_EeJOk)which explains it a bit

**Comment by pattersonbl2** [comment]
Are you still needing help on this issue ?

**Reply by blairmacintyre** [reply]
I am.   I just encountered this.  I uploaded a bunch of avatars (readyplayer.me avatars) by creating them in the avatar picker.  Went to admin and approved/promoted them.  And now when I go to a room I get the default grey block avatar, and if I try to change it, I see now avatars in any of the tabs.  Yikes!

**Reply by blairmacintyre** [reply]
ok, I think my problem is that I edited one of the attribution fields, and whatever code is responsible for taking that attribution and putting it in the database is doing it wrong (i.e., perhaps it's putting an escaped string in instead of just the raw json).  Regardless, errors on the console imply this.  

e.g., 
Oct 01 17:34:33 confident-thief.realitymedia-internal.net bash[918]: reticulum.default@realitymedia-hub(O):     ** (ArgumentError) cannot load `"{\"creator\":\"readyplayer.me\"}"` as type :map for field :attributions in %Ret.Avatar{bin_owned_file_id: nil, orm_map_owned_file_id: nil, imported_from_port: nil, account: #Ecto.Association.NotLoaded<association :account is not loaded>, emissive_map_owned_file: #Ecto.Association.NotLoaded<association :emissive_map_owned_file is not loaded>, parent_avatar_id: nil, avatar_sid: nil, bin_owned_file: #Ecto.Association.NotLoaded<association :bin_owned_file is not loaded>, gltf_owned_file: #Ecto.Association.NotLoaded<association :gltf_owned_file is not loaded>, description: nil, gltf_owned_file_id: nil, slug: nil, avatar_listings: #Ecto.Association.NotLoaded<association :avatar_listings is not loaded>, state: nil, allow_remixing: nil, parent_avatar_listing: #Ecto.Association.NotLoaded<association :parent_avatar_listing is not loaded>, attributions: nil, base_map_owned_file_id: nil, account_id: nil, allow_promotion: nil, parent_avatar: #Ecto.Association.NotLoaded<association :parent_avatar is not loaded>, emissive_map_owned_file_id: nil, imported_from_host: nil, inserted_at: nil, base_map_owned_file: #Ecto.Association.NotLoaded<association :base_map_owned_file is not loaded>, parent_avatar_listing_id: nil, normal_map_owned_file: #Ecto.Association.NotLoaded<association :normal_map_owned_file is not loaded>, updated_at: nil, thumbnail_owned_file: #Ecto.Association.NotLoaded<association :thumbnail_owned_file is not loaded>, reviewed_at: nil, __meta__: #Ecto.Schema.Metadata<:loaded, "ret0", "avatars">, orm_map_owned_file: #Ecto.Association.NotLoaded<association :orm_map_owned_file is not loaded>, imported_from_sid: nil, avatar_id: nil, thumbnail_owned_file_id: nil, name: nil, normal_map_owned_file_id: nil}
